Translations/ru.po

```diff
@@ -5062,10 +5066,12 @@ msgctxt "monster"
 msgid "Madeye the Dead"
 msgstr "Бесоглаз Мертвец"
 
+# https://en.wikipedia.org/wiki/Chupacabra
+# "El" = "The".
 #: Source/monstdat.cpp:499
 msgctxt "monster"
 msgid "El Chupacabras"
-msgstr "Чупакабры"
+msgstr "Чупакабра"
```

Comment:
Is this plural or singular? I think this is a single monster, in which this is good.

Reply:
arbitrary